### PR TITLE
[BACK-2963] Standard Deviation and other new CGM stats

### PR DESCRIPTION
**Pull request by Roukoswarf**

### CODE REVIEW

File: reference/summary/models/continuous/continuousstats.v1.yaml [hunk 1]
```diff
@@ -0,0 +1,21 @@
+type: object
+title: continuousstats.v1
+description: 'A summary of a users recent CGM glucose values'
```
Comment (ewollesen) [inline reference/summary/models/continuous/continuousstats.v1.yaml:3]:
```suggestion
description: A summary of a user's recent CGM glucose values
```

**Review by ewollesen — APPROVED** [review]
Just one typo fix, but nothing to block you.

File: reference/clinic.v1.yaml [hunk 2]
```diff
@@ -364,13 +378,83 @@ paths:
...
           in: query
           name: cgm.timeInAnyHighPercent
           description: Percentage of time above 180 mg/dL
+        - schema:
+            type: string
+            pattern: ^(>=|>|<=|<)\d\.\d\d?$
+            example: '>=0.5'
+          in: query
+          name: cgm.timeCGMUseMinutes
+          description: Minutesage of time of CGM use
```
[the remark this answers is not in the document]
Comment (Roukoswarf) [reply reference/clinic.v1.yaml:401]: regex typo when adding delta filtering

File: reference/clinic.v1.yaml [hunk 3]
```diff
@@ -565,6 +679,385 @@ paths:
...
+          in: query
+          name: cgm.glucoseManagementIndicatorDelta
+          description: Glucose management Indicator of records
+        - schema:
+            type: string
+            pattern: ^(>=|>|<=|<)\d\.\d\d?$
+            example: '>=0.5'
+          in: query
+          name: cgm.timeCGMUsePercentDelta
+          description: PercentDeltaage of time of CGM use
```
[the remark this answers is not in the document]
Comment (Roukoswarf) [reply reference/clinic.v1.yaml:703]: regex typo, fixed.

File: reference/clinic.v1.yaml [hunk 4]
```diff
@@ -565,6 +679,385 @@ paths:
...
           description: Total records in period
+
+        - schema:
+            type: string
+            pattern: ^(>=|>|<=|<)\d\.\d\d?$
+            example: '>=5'
+          in: query
+          name: cgm.averageGlucoseMmolDelta
+          description: Average glucose value of records
+        - schema:
```
[the remark this answers is not in the document]
Comment (ewollesen) [reply reference/clinic.v1.yaml:690]: Oh, wow, I must have brain faeded... the name includes Mmol. I was only looking at the description. My bad. Withdrawn.

File: reference/clinic.v1.yaml [hunk 5]
```diff
@@ -316,608 +316,436 @@ paths:
...
+          description: Percentage of time [0.0-1.0]  above 250 mg/dL
         - schema:
-            type: string
-            pattern: ^(>=|>|<=|<)\d\.\d\d?$
-            example: '>=0.7'
+            $ref: '#/components/schemas/FloatFilter'
           in: query
           name: bgm.timeInExtremeHighPercent
-          description: Percentage of time above 350 mg/dL
+          description: Percentage of time [0.0-1.0]  above 350 mg/dL
```
Comment (ewollesen) [inline reference/clinic.v1.yaml:527]:
```suggestion
          description: Percentage of time [0.0-1.0] above 350 mg/dL
```
Consider removing the extra space the next time you're making changes here.

**Review by ewollesen — APPROVED** [review]

**Pull request merged**